src/util.c

```diff
@@ -19,6 +19,28 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 
 #include <fontconfig/fontconfig.h>
 
+char* load_img_stdin(size_t *lenout){
```

Comment:
Opening brace on newline for functions please. Putting it on a newline helps separate the parameters from the body of the function.